fdbserver/TLogServer.actor.cpp

```diff
@@ -2408,11 +2412,13 @@ ACTOR Future<Void> tLogCommit(TLogData* self,
 		// Notifies the commitQueue actor to commit persistentQueue, and also unblocks tLogPeekMessages actors
 		logData->version.set(req.version);
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
-			self->unknownCommittedVersions.push_front(std::make_tuple(req.version, req.tLogCount));
+			self->unknownCommittedVersions.push_front(std::make_tuple(req.version, req.seqPrevVersion, req.tLogLocIds));
 			while (!self->unknownCommittedVersions.empty() &&
 			       std::get<0>(self->unknownCommittedVersions.back()) <= req.knownCommittedVersion) {
 				self->unknownCommittedVersions.pop_back();
 			}
+		} else {
+			ASSERT(req.prevVersion == req.seqPrevVersion); // @todo remove this assert later
```

Comment:
I think the transaction system would stall if this was not true in tlogCommit, you could remove.

Reply:
I will keep it for now, and will remove it in a follow up PR.